src/services/ubi/community.ts

```diff
@@ -643,14 +653,30 @@ export default class CommunityService {
             ],
             where: {
                 communityId: community.publicId,
-                active: true,
+                ...activeCondition,
             },
         });
+
+        const address = result.map((el) => `'${el.address}'`);
+        const beneficiariesAdded:
+            | { count: number; from: string }[]
+            | undefined = await this.sequelize.query(
+            `SELECT count("address"), "from" FROM ubi_beneficiary_registry WHERE "from" IN (${address.toString()}) GROUP BY "from"`,
+            {
+                raw: true,
+                type: QueryTypes.SELECT,
+            }
+        );
```

Comment:
If we can avoid raw queries, lets do it. Keeping the code clean and consistent.
Follow this example https://stackoverflow.com/a/66393583/3348623 something like a .findAll with `raw:true` and `fn count` generates de same query, and it's easier to read.

Reply:
done!